packages/ethereum-contracts/contracts/agreements/ConstantFlowAgreementV1.sol

```diff
@@ -813,6 +813,88 @@ contract ConstantFlowAgreementV1 is
         emit FlowOperatorUpdated(token, currentContext.msgSender, flowOperator, permissions, flowRateAllowance);
     }
 
+    /// @dev IConstantFlowAgreementV1.increaseFlowRateAllowance implementation
+    function increaseFlowRateAllowance(
+        ISuperfluidToken token,
+        address flowOperator,
+        int96 flowRateAllowanceDelta, // flowRateBudget
```

Comment:
1) change to substracted/addedFlwoRateAllowance makes ense.
2) should stick to int96 and consider it is our "custom type for flowrate" befor custom type is a thing in solidity; but of course continue to range check here before using it.

Reply:
using uint48 would not be sufficient:
uint48 max value is: `281474976710655`
int96 max value is: `39614081257132168796771975167`.
We can 
I can rename the `delta` params though, it will remove any possible confusion given the int param